package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "labs-ui",
-  "version": "0.0.26",
+  "version": "1.0.0",
```

Comment:
:champagne: 

@andycochran for upgrading existing apps to use v1, is the main thing we need to change to make sure we install the peer dependencies alongside labs-ui@1.0.0?

Reply:
To upgrade an existing app that use Labs UI to v1, in addition to making sure dependencies are installed (all likely already are except the style guide), we need to also reconfigure how its Sass is set up and make sure that any of the consuming app's custom components that depended on Labs UI classes for styling are updated to work with the style guide's classes. Specifically, the color palette and margin/paddding classes that were old Labs UI has change in the style guide. Checking/updating these classes should be as simple as using the app, noting any bad color/margin/padding, and updating the classes in the markup.

Reply:
@bfreeds Did you have concerns that the version should be 0.1.0 instead of 1.0.0? I'd consider this a major bump, not minor or patch. Right?